core/src/validator.rs

```diff
@@ -368,6 +370,8 @@ impl ValidatorConfig {
             replay_forks_threads: NonZeroUsize::new(1).expect("1 is non-zero"),
             replay_transactions_threads: NonZeroUsize::new(get_max_thread_count())
                 .expect("thread count is non-zero"),
+            tvu_shred_sigverify_threads: NonZeroUsize::new(get_thread_count())
+                .expect("thread count is non-zero"),
```

Comment:
Having to duplicate this (in addition to calling the function in `validator/cli/thread_args.rs` is a bit unfortunate, but it was what we did with `replay_transactions_threads`.

For now, I think I'm ok with it. Eventually, I'd like the `solana_rayon_threadlimit` crate to go away completely, removing these "extraneous" usages could be tucked in with that